docs/content/zh/docs/examples/icmp-remote.md

```
@@ -0,0 +1,27 @@
+
+## 命令
+
+```bash
+sudo ptcpdump -i any 'icmp and host 1.1.1.1'
+```
+
+在捕获处于活动状态时运行 `ping -w 10 -c 2 1.1.1.1`。期望看到记录了 `ping` 命令的回显请求，以及一个可以重放以确认元数据持久存在的 pcapng 文件。
+
+## 输出示例
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)